pkg/scenario/io.go

```diff
@@ -16,6 +15,7 @@ import (
 
 const (
 	scenarioDir = "scenario"
+	scenSize    = 1 << 30 // 1Gb
```

Comment:
This limit is arbitrary. I don't expect a scenario to be this large thus I avoid zip bombs.